refocus-bdk-server.js

```diff
@@ -278,23 +278,24 @@ module.exports = (config) => {
    *
    * @param {Express} app - App stream so we can push events to the server
    * @param {String} token - Socket Token needed to connect to Refocus socket
+   * @param {String} botId - Id of the bot
    */
-  function refocusConnectSocket(app, token) {
+  function refocusConnectSocket(app, token, botId) {
     const opts = {
       reconnect: true,
       'reconnection delay': 10,
       transports: ['websocket'],
       upgrade: false,
-      extraHeaders: {
-        Authorization: token
-      }
+      query: {
```

Comment:
does this mean the bot will only recieve messages directed to it?

Reply:
That's the idea. Is that right?